test/test_complex.py

```diff
@@ -9,5 +9,62 @@ def test_to_list_with_complex_64(self):
         # there's no garbage value in the resultant list
         self.assertEqual(torch.zeros((2, 2), dtype=torch.complex64).tolist(), [[0j, 0j], [0j, 0j]])
 
+    def test_exp_with_complex_64(self):
+        import numpy as np
+
+        w = torch.tensor([0 + 0j, 1 + 1j, 2 + 2j, 3 + 3j], dtype=torch.complex64)
+        w = torch.exp(w)
+
+        x = np.array([0 + 0j, 1 + 1j, 2 + 2j, 3 + 3j], dtype=np.complex64)
+        x = np.exp(x)
+
+        self.assertEqual(w * 2j, x * 2j)
+
+    def test_exp_with_complex_128(self):
+        import numpy as np
+
+        w = torch.tensor([0 + 0j, 1 + 1j, 2 + 2j, 3 + 3j], dtype=torch.complex128)
+        w = torch.exp(w)
+
+        x = np.array([0 + 0j, 1 + 1j, 2 + 2j, 3 + 3j], dtype=np.complex128)
+        x = np.exp(x)
+
+        self.assertEqual(w * 2j, x * 2j)
+
+    def test_addition_and_mult_with_complex_64(self):
```

Comment:
I don't think we need ```test_addition_and_mult_with_complex_64```,  ```test_exp_and_addition_with_complex_64```, and ```test_exp_and_addition_with_complex_128``` because they are different from the first test only in the way that we test the addition of complex tensor and a scalar to output a complex tensor :D